lib/utils/messages_test.go

```diff
@@ -25,9 +25,20 @@ import (
 
 func TestMessageError(t *testing.T) {
 	message := Message{
-		Code: "ProvisioningFailed",
-		Type: "Invalid",
+		Code:        "ProvisioningFailed",
+		Type:        "Invalid",
+		Description: "Failed to create file share with the storage provider",
```

Comment:
let's have generic name not specific to file

Reply:
As discussed @arahamad 
Removed the test cases all together